### PR TITLE
feat:bag-page

### PR DESCRIPTION
**Pull request by TheAmlanSaikia**

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
nice work

File: Cart/Cart.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+.main-container {
+    margin: var(--value-3) auto 0 auto;
+    grid-area: main;
+    width: 68%;
+    height: max-content;
+    display: flex;
+
+
```
Comment (rajkishancode) [inline Cart/Cart.css]: remove extra space

File: Cart/Cart.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                                Discount
+                            </p>
+                            <p class="value">
+                                ₹ 0.00
+                            </p>
+                        </div>
+
+                        <div class="mrp flex-row justify-between mt-1 checkout-row">
+                            <p class="text-md">
+                                Convience Fee
```
Comment (rajkishancode) [inline Cart/Cart.html]: typo "Convience".Please  correct this

**Review by rajkishancode — APPROVED** [review]
Left few comments  and looks good to merge.

**Pull request merged**